src/keybindings.c

```diff
@@ -1827,11 +1827,11 @@ static void goto_matching_brace(GeanyDocument *doc)
 
 static gboolean cb_func_clipboard_action(guint key_id)
 {
-	GeanyDocument *doc = document_get_current();
+	GtkWidget *focusw = gtk_window_get_focus(GTK_WINDOW(main_widgets.window));
 
-	if (doc == NULL)
+	if (!IS_SCINTILLA(focusw)) {
```

Comment:
@b4n They don't work for me in places I would like currently (i.e. the file viewer pane). The previous code has the same properties (except it also returned if focusw targeted a splitwindow because that would have a different sci object from doc->editor->sci). 
```if (doc == NULL || focusw != GTK_WIDGET(doc->editor->sci)) ```